packages/components/tests/dummy/app/templates/components/tabs.hbs

```diff
@@ -284,10 +280,10 @@
       <div class="dummy-tab-base-sample-item">
         <span class="dummy-text-small">{{capitalize state}}:</span>
         <ul class="dummy-tab-ul-tab-wrapper">
-          <Hds::Tabs::Tab mock-state-value={{state}} mock-state-selector="a">Lorem ipsum</Hds::Tabs::Tab>
+          <Hds::Tabs::Tab mock-state-value={{state}} mock-state-selector="*">Lorem ipsum</Hds::Tabs::Tab>
         </ul>
         <ul class="dummy-tab-ul-tab-wrapper">
-          <Hds::Tabs::Tab @icon="hexagon" @count="10" mock-state-value={{state}} mock-state-selector="a">Lorem ipsum</Hds::Tabs::Tab>
+          <Hds::Tabs::Tab @icon="hexagon" @count="10" mock-state-value={{state}} mock-state-selector="*">Lorem ipsum</Hds::Tabs::Tab>
```

Comment:
The Tab DOM was refactored to use a button element instead of a link. The Icon and Count had to be moved outside of the button element to fix accessibility issues the linter complained about. In order to change the icon color on hover as well as the button color, they both needed to use a mock-hover class. (I couldn't figure out how to add it the parent wrapper component instead, using "li" as the selector didn't work. If there's a better way to do this please let me know.)

Reply:
@KristinLBradley I have pushed a commit (https://github.com/hashicorp/design-system/pull/607/commits/b34dc98a11959cc21663daf17b4191d87590ea9c) that fixes this. Have a look at my changes and let me know if something is not clear (in this case, it's slightly different from other components, because the elements that receive the `hover` and the `focus` states are different).